src/validate/index.ts

```diff
@@ -0,0 +1,11 @@
+import { asset } from "./methods/asset";
```

Comment:
All the validation methods will go here to make them easier to find.

Reply:
very nice! I'll use this once it's merged for Fund account